### PR TITLE
Clean Back-office templates, part 1 - common

### PR DESCRIPTION
**Pull request by Hlavtox**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | See below.
| Type?             | improvement
| Category?         | BO
| BC breaks?        | This PR should be considered as a BC because of lot of HTML structure changes and they might be used by a module or anything else, but it's acceptable. The optimizations are only within the files itself, so it does not matter if anyone uses the old version in any form in their modules.
| Deprecations?     | no
| How to test?      | Briefly go through pages that implement Grids (like order list) and see if nothing is broken by this PR. There should be no visible change.
| Possible impacts? | No bad impacts. :-)

### Description
- I went through all templates and removed 99% of unneeded boostrap code like `<div class="row"><div class="col">XXX</div></div>`. It will reduce render depth and complexity by several layers - easier to navigate, debug CSS, maybe faster to render on slower PCs.
- I fixed all places where somebody used single quotations or no quotations at all.
- I fixed all places where somebody combined row and column in same element, this is not allowed.
- I fixed all places where somebody used syntax line col-sm-12 col-md-12 or col-md-12 col-lg-6. The first one is not needed.
- Small fixes here and there - removed some container classes, fixed module position page responsivity a bit, fixed virtual product block responsivity etc...
It should produce no BC breaks, unless some 3rd party module was targetting some classes they shouldn't have OR they targeted it by child or parent.

### Templates concerned
Legacy wrapper
Common
Common/Grid
Exception

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26060)

### CODE REVIEW

**Review by NeOMakinG — APPROVED** [review]
Nice!

@Julievrz There are no wording changes, it got labelised because of markup changes

File: src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/bootstrap_4_layout.html.twig [hunk 1]
```diff
@@ -827,7 +827,7 @@
 
 {% block form_hint %}
   {% if hint %}
-    <div class="row col-lg hint-box">
+    <div class="hint-box">
```
Comment (jolelievre) [inline src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/bootstrap_4_layout.html.twig:830]: Just a comment about this one, we will stop using this bootstrap 4 layout soon. The new prestashop UI kit extends the layout from Symfony framework See here:
https://github.com/PrestaShop/PrestaShop/blob/83977178c98e95be1ca6b7393d16a40766feb8c5/src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit_base.html.twig#L48

Once all the forms will be migrated to the new UI kit form theme, we won't use this file at all, so I hope your modifications will still work as expected when everything is cleaned. Maybe we'll have to override some parts of the template to match your changes?

File: src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/form_div_layout.html.twig [hunk 2]
```diff
@@ -315,7 +314,10 @@
     {% set form_method = "POST" %}
   {%- endif -%}
   <form name="{{ name }}"
-        method="{{ form_method|lower }}" action="{{ action }}"{% for attrname, attrvalue in attr %} {{ attrname }}="{{ attrvalue }}"{% endfor %}{% if multipart %} enctype="multipart/form-data"{% endif %}>
+        method="{{ form_method|lower }}"
```
Comment (jolelievre) [inline src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/form_div_layout.html.twig:317]: Same comment as bootstrap4 layout, this file will stop being used and probably will be removed.

**Review by jolelievre — APPROVED** [review]
Thanks @Hlavtox This will clean the BO a lot, I just added two comments to be careful about the parent layout. These files will not be used by the new UI kit form theme so I hope their modifications are not mandatory.

If they are I guess we will be able to fix this later when we clean everything, once all forms will have been updated to use the new form theme.

File: src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/bootstrap_4_layout.html.twig (hunk 1 above)
Comment (Hlavtox) [reply src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/bootstrap_4_layout.html.twig:830]: @jolelievre Hi! Should not matter. Everything will work as expected. ;-)

Even if anybody uses or overrides only one file of my PR, or half, or everything, it has no effect on other files. Everything will look and function completely the same.

**Comment by Julievrz** [comment]
> Nice!
> 
> @Julievrz There are no wording changes, it got labelised because of markup changes

Thanks, I'm removing the label but feel free to put it back or ping me if needed.

**Review by PierreRambaud — APPROVED** [review]

**Comment by hibatallahAouadni** [comment]
Hello @Hlavtox 

Sorry for the delay but I need to test it thoroughly (and I launched the automated tests) and it seems all good.
So, LGTM and QA :heavy_check_mark: 

Thanks!

**Comment by Progi1984** [comment]
Thanks @Hlavtox & @hibatallahAouadni

**Pull request merged**